client/signup/steps/theme-selection/index.jsx

```diff
@@ -101,42 +123,27 @@ class ThemeSelectionStep extends Component {
 
 	render = () => {
 		const defaultDependencies = this.props.useHeadstart ? { themeSlugWithRepo: 'pub/twentysixteen' } : undefined;
+		const { translate } = this.props;
 
-		const pressableWrapperClassName = classNames( {
-			'theme-selection__pressable-wrapper': true,
-			'is-hidden': ! this.state.showPressable,
-		} );
-
-		const themesWrapperClassName = classNames( {
-			'theme-selection__themes-wrapper': true,
-			'is-hidden': this.state.showPressable,
-		} );
+		let headerText = translate( 'Choose a theme.' );
```

Comment:
I did it with the shorthand `if` syntax and `const`, but didn't look very nice and wasn't very obvious what was happening there:
```
const headerText = this.state.showPressable
	? translate( 'Upload your WordPress Theme' )
	: translate( 'Choose a theme.' );

const subHeaderText = this.state.showPressable
	? translate( 'Our partner Pressable is here to help you' )
	: translate( 'No need to overthink it. You can always switch to a different theme later.' );
```

versus:

```
let headerText = translate( 'Choose a theme.' );
let subHeaderText = translate( 'No need to overthink it. You can always switch to a different theme later.' );

if ( this.state.showPressable ) {
	headerText = translate( 'Upload your WordPress Theme' );
	subHeaderText = translate( 'Our partner Pressable is here to help you' );
}
```

I think the second one is better for readability if you're quickly skimming through the code and I don't think `let`/`const` would make a huge difference in terms of performance here :)